pom.xml

```diff
@@ -257,11 +257,6 @@
               <configuration>
                 <!-- Sets the path to the file which contains the execution data. -->
                 <destFile>${project.build.directory}/coverage-reports/jacoco-ut.exec</destFile>
-                <!--
```

Comment:
What is the reason for this change ? Is this related to test cases ?

Reply:
This is an unrelated change but is required to run jacoco. Currently jacococ plugin does not run on master branch.

I will separate this into another PR.